[DOCS-15097] Subqueries & Reference Table joins in log monitors (#38155)
DataDog/documentation: content/en/monitors/types/log.md

   * **Third facet**: 5 top values
   * **Fourth facet**: 2 top values

4. Modify the query with additional options (optional):

>    Use the {{< ui >}}Σ Modify Query{{< /ui >}} menu to refine the query with:

Reviewer: ```suggestion
4. Optionally, use the {{< ui >}}Σ Modify Query{{< /ui >}} menu to refine the query with:
```

Reviewer: Suggesting a minor rephrasing so that the step leads with the UI action instead of showing it below. Happy to reevaluate if you have a different perspective.

   * **Functions and formulas**: Transform and combine your query results. See [functions][11] and [formulas][12].

>    * **Advanced search**: Scope alerts to curated datasets, such as entity lists, customer tiers, or correlated events. See [subqueries and Reference Table joins][13].

Reviewer: ```suggestion
   * **Advanced search**: Scope alerts to curated datasets, such as entity lists, customer tiers, or correlated events. See [subqueries][13] and [Reference Tables][14].
```

Reviewer: Per the [style guide on link text](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md#links), suggesting to update this so that each goes to its respective section. Otherwise, the user has to scroll to it when landing on the Advanced Search page. I'm open to evaluating other options if you think something else would be better.

[8]: /monitors/notify/
[9]: /monitors/notify/variables/?tab=is_alert#matching-attributetag-variables
[10]: /logs/log_configuration/flex_logs/
[11]: /logs/explorer/analytics/#functions
[12]: /logs/explorer/analytics/#formulas

> [13]: /logs/explorer/advanced_search/

Reviewer: ```suggestion
[13]: /logs/explorer/advanced_search/#filter-logs-with-subqueries
[14]: /logs/explorer/advanced_search/#filter-logs-based-on-reference-tables
```
